.eslintrc.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
     ecmaVersion: 2016,
     sourceType: 'module'
   },
+  globals: {
+    dataProvider: 'readonly',
+    testData: 'readonly'
```

Comment:
we should add this to our eslint package

Reply:
I have removed globals from this file and moved it to eslint module.